src/lib/bootloader/kexec.rb

```diff
@@ -99,8 +99,8 @@ def copy_kernel
       cmd = Yast::Builtins.sformat(
         "/bin/cp -L %1/%2 %1/%3 %4",
         Yast::Installation.destdir,
-        "vmlinuz",
-        "initrd",
+        "boot/vmlinuz",
+        "boot/initrd",
```

Comment:
you are right, it already have test, just it do not check exact location, see - https://github.com/yast/yast-bootloader/blob/master/test/kexec_test.rb#L87
I am not sure if checking exact location helps there, as it is single shot fix and can change in future ( I expect that vmlinuz symlink is arch specific, so it can be more tricky and we can get some bug reports, if SLE start using this kexec ).